src/app/components/jamjarPlayer/jamjarPlayer.scss

```diff
@@ -93,6 +93,11 @@ videogular div.jamjar-layer {
     //transition: all 1s ease-out;  
 }
 
+.jamjar-line-text {
+  line-height: 30px;
+
+}
+

```

Comment:
as discussed, we can take out "Now Playing" from the bars. Also, for the .jamjar-line class, can we change the height to 15px and add opacity to 0.5? this way the user can see the video behind better